### PR TITLE
[Prim] Add op map

### PR DESCRIPTION
**Pull request by cyber-pioneer**
### PR types
Others

### PR changes
Others

### Describe
1. Add op map of composite ops flatten
2. remove unused all list

### CODE REVIEW

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

File: paddle/phi/api/yaml/op_compat.yaml [hunk 1]
```diff
@@ -594,6 +594,14 @@
   outputs :
     out : Out
 
+- op : flatten (flatten_contiguous_range)
+  inputs :
+    x : X
+  outputs :
+    {out : Out, xshape : XShape}
+  attrs :
+    {start_axis : start_axis, stop_axis : stop_axis}
```
Comment (zyfncg) [inline paddle/phi/api/yaml/op_compat.yaml:603]: 命名相同参数的不用再加映射

File: paddle/phi/api/yaml/op_compat.yaml [hunk 2]
```diff
@@ -648,6 +656,14 @@
   extra :
     attrs : [bool use_mkldnn = false]
 
+- op : full_like (fill_any_like)
+  inputs :
+    x : X
+  outputs :
+    out : Out
+  attrs :
+    {value: value, dtype: dtype}
```
Comment (zyfncg) [inline paddle/phi/api/yaml/op_compat.yaml:665]: 同上

File: paddle/phi/api/yaml/op_compat.yaml (hunk 1 above)
Comment (cyber-pioneer) [reply paddle/phi/api/yaml/op_compat.yaml:603]: ok，后续去掉

**Review by zyfncg — APPROVED** [review]

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**